src/wiktextract/pronunciations.py

```diff
@@ -582,72 +582,72 @@ def split_cleaned_node_on_newlines(contents):
         # XXX what about {{hyphenation|...}}, {{hyph|...}}
         # and those used to be stored under "hyphenation"
 
-        # Add data that was collected in template_fn
-        if audios:
```

Comment:
`if` could be removed, the `for` loop won't run when `audios` is empty. The problem of this code is this function is too long and has too many nested functions. It's hard not to make indent mistakes or override shared variables in code like this.